lib/main/STM32F7/Drivers/STM32F7xx_HAL_Driver/Src/stm32f7xx_hal.c

```diff
@@ -154,10 +154,14 @@ __IO uint32_t uwTick;
 HAL_StatusTypeDef HAL_Init(void)
 {
   /* Configure Flash prefetch and Instruction cache through ART accelerator */ 
-#if (ART_ACCLERATOR_ENABLE != 0)
+#if (ART_ACCLERATOR_ENABLE != 0U)
    __HAL_FLASH_ART_ENABLE();
 #endif /* ART_ACCLERATOR_ENABLE */
 
+#if (PREFETCH_ENABLE != 0U)
```

Comment:
These changes in `lib/` are running the risk of being overwritten if the libraries get updated. Is there a way to achieve the same with changes in `src/main`?

Reply:
I'm not entirely sure why it wasn't there to begin with, it should have